src/lib/toast/Toast.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+	import { toasts, removeToast } from './toastStore';
+	import { Button } from '$lib';
+	import type { SvelteHTMLElements } from 'svelte/elements';
+	import { twMerge } from 'tailwind-merge';
+
+	function generateVariantStyle(color: string) {
+		return `animate-slideIn items-center text-${color}-800 border border-${color}-300 bg-${color}-50 dark:bg-gray-800 dark:text-${color}-400 dark:border-${color}-300 my-4 flex p-4 rounded-lg`;
```

Comment:
Tailwind doesn't allow for interpolation of values - See https://tailwindcss.com/docs/content-configuration#dynamic-class-names